cern_access/indico_cern_access/util.py

```diff
@@ -284,16 +284,16 @@ def grant_access(registrations, regform):
                                  reg.cern_access_request.request_state == CERNAccessRequestState.active)]
     state, data = send_adams_post_request(event, new_registrations)
     add_access_requests(new_registrations, data, state)
-    send_form_link(event, new_registrations)
+    send_form_link(event, new_registrations, regform)
 
 
-def send_form_link(event, registrations):
+def send_form_link(event, registrations, regform):
     """Send a mail asking for personal information to be filled in using a web form."""
     start_dt, end_dt = get_access_dates(get_last_request(event))
-
+    email_ticket = regform.ticket_on_email
```

Comment:
you could get this from the registration object instead of adding a new argument to pass the regform